README.md

```diff
@@ -1,101 +1,482 @@
-## loopback-connector-oracle
+# loopback-connector-oracle
 
-The Oracle Connector module for for [loopback-datasource-juggler](https://github.com/strongloop/loopback-datasource-juggler).
+[Oracle](https://www.oracle.com/database/index.html) is an object-relational database management system produced by Oracle Corporation. The `loopback-connector-oracle` module is the Oracle connector for the LoopBack framework.
 
-For full documentation, see the official [Documentation](http://loopback.io/doc/en/lb2/Oracle-connector.html).
+For more information, see the [LoopBack documentation](http://loopback.io/doc/en/lb3/Oracle-connector.html).
 
-**NOTE**: This connector is compatible with Node v.0.10.x, v.12.x, and iojs v1.6.2.  
+## Prerequisites
+
+**Node.js**: The Oracle connector is compatible with Node v.0.10.x, v.12.x, and later.  
+
+**MacOS**: The Oracle connector does not work on Mac OSX 10.11 (El Capitan) due to [SIP](https://en.wikipedia.org/wiki/System_Integrity_Protection) runtime protections and restrictions.
```

Comment:
If this is for the new version that [uses node-oracledb](https://github.com/strongloop/loopback-connector-oracle/blob/master/lib/oracle.js#L12) as the underlying DB access add-on, then it is incorrect to say it doesn't work on 10.11.  It does work. See https://github.com/oracle/node-oracledb/blob/master/INSTALL.md#instosx